artifacthub/library/general/allowedrepos/1.0.0/artifacthub-pkg.yml

```diff
@@ -3,7 +3,7 @@ name: k8sallowedrepos
 displayName: Allowed Repositories
 createdAt: "2022-09-26T17:28:27Z"
 description: Requires container images to begin with a string from the specified list.
-digest: eaec68f7273f5d79ced3ed778d5c2fc26b241e5db3a0e5119650bab651f8f1ed
```

Comment:
Hi @anderseknert - I suspect these previous version changes are likely "left over" from the generate prior to the template's patch version bump.  An option might be to drop the changes from `artifacthub/library` and then re-run `make generate-all`.

Reply:
Thanks, I'll give it a shot!

Reply:
Thanks - Those folders look good now!